lib/data-types.js

```diff
@@ -562,6 +562,76 @@ DATE.prototype._stringify = function _stringify(date, options) {
   return date.format('YYYY-MM-DD HH:mm:ss.SSS Z');
 };
 
+/**
+ * Date column with timezone, default is UTC
+ *
+ * @namespace DataTypes.SMALLDATETIME
+ */
+function SMALLDATETIME(length) {
```

Comment:
This looks like 1:1 copy paste from the regular date type, some abstraction would help.